services/orchest-webserver/app/app/views/views.py

```diff
@@ -1069,6 +1070,15 @@ def filemanager_read():
                 raise app_error.OutOfDataDirectoryError(
                     "Path points outside of the data directory."
                 )
+        elif path.endswith("schema.json"):
```

Comment:
Nit (non-blocking): Maybe something like this is better:

```suggestion
        elif path.endswith(".schema.json"):
```

Just to protect against false positives (albeit highly unlikely).

Reply:
I changed it as follows:

```
JSON_SCHEMA_FILE_EXTENSIONS = [".schema.json", ".uischema.json"]
...
elif any(path.endswith(extension) for extension in JSON_SCHEMA_FILE_EXTENSIONS):
```
It was meant for `.schema.json` and `.uischema.json`.